### ISSUE TITLE
Missing icons

### ISSUE THREAD

**Issue by Dudi00** [bhilburn/powerlevel9k#245]
Hi I'm trying to use [this config](https://github.com/bhilburn/powerlevel9k/wiki/Show-Off-Your-Config#rjorgensons-configuration), I've installed awesome fonts and set them as primary font. 

[my current status](http://i.imgur.com/3mG1iZ3.png)

As you can see some icons/charcters are missing. I think I've done everythink correct, what I'm missing?

**Comment by dritter** [comment]
A quick guess:
Did you set the `POWERLEVEL9K_MODE='awesome-patched'`? And did you set this value it _before_ applying/sourcing the theme (in your ZSH framework)?

**Comment by Dudi00** [comment]
Ooo... before applying/sourcing the theme is the key.
Thanks @dritter :+1:

**Comment by guidoilbaldo** [comment]
My current situation:
![screen shot 2017-06-08 at 11 44 47](https://user-images.githubusercontent.com/6077718/26922777-6489f862-4c40-11e7-9fb3-fef52402badd.png)

I've installed awesome regular fonts as showed here:
![screen shot 2017-06-08 at 11 45 05](https://user-images.githubusercontent.com/6077718/26922791-73a387f0-4c40-11e7-9e4d-c819b5d0958f.png)

and my .zshrc file is configured as follows:
![screen shot 2017-06-08 at 11 46 00](https://user-images.githubusercontent.com/6077718/26922808-7eba2306-4c40-11e7-9a9a-43b8c9f3da78.png)

Any help would be much appreciated!

**Comment by dritter** [comment]
@guidoilbaldo You are using the wrong mode. 😉 
`14pt SourceCodePro+Powerline+Awesome Regular` is a pre patched font. So you need to set `POWERLEVEL9K_MODE="awesome-patched"` to make the font work.

**Comment by guidoilbaldo** [comment]
Yeah I got it working yesterday, my damn fault putting confs together from different browser tabs :) thanks for answering tho!

**Comment by bhilburn** [comment]
Glad you're all set, @guidoilbaldo! Enjoy =)

**Comment by tsouhaieb** [comment]
for example if you are using prezto like me, 
add POWERLEVEL9K_MODE='awesome-patched'
before 
if [[ -s "${ZDOTDIR:-$HOME}/.zprezto/init.zsh" ]]; then
  source "${ZDOTDIR:-$HOME}/.zprezto/init.zsh"
fi

your code will be than 
```
POWERLEVEL9K_MODE='awesome-patched'# Source Prezto.
if [[ -s "${ZDOTDIR:-$HOME}/.zprezto/init.zsh" ]]; then
  source "${ZDOTDIR:-$HOME}/.zprezto/init.zsh"
fi
```
NB: don't forget to use a patched font in your terminal for EXP 
source code pro powerline awesome regular
#

**Comment by AzySir** [comment]
@dritter @guidoilbaldo 
My config file isn't as complex as @guidoilbaldo's config - I must have missed a whole chunk of set up instructions. 
I've got the powerlevel9k theme working but still getting the question marks.

I've set to the same fonts as @guidoilbaldo in the hope that may fix it but no luck. 

<img width="557" alt="screen shot 2019-03-08 at 10 03 52 am" src="https://user-images.githubusercontent.com/37623164/53995573-8d645a80-4189-11e9-8a3c-143c5eb6fd05.png">

<img width="525" alt="screen shot 2019-03-08 at 10 03 28 am" src="https://user-images.githubusercontent.com/37623164/53995585-97865900-4189-11e9-8c3f-d67b873d47ef.png">
